### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by alexdamrow**
Please review @HenevianAcero

### CODE REVIEW

File: prac_07/myguitars.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    in_file.close()
+
+
+def save_file(guitars):
+    """Save a file."""
+    out_file = open("guitars.csv", 'w')
+    for guitar in guitars:
+        print(f"{guitar.name},{guitar.year},{guitar.cost}", file=out_file)
+
+
```
Comment (HenevianAcero) [inline prac_07/myguitars.py:44]: Good work

File: prac_07/programming_language.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Programming Language class with tests.
+"""
```
Comment (HenevianAcero) [inline prac_07/programming_language.py:4]: Missing estimated/actual time

File: prac_07/project.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+"""
+CP1404 Prac 7 - Project management program
+Estimated time: 2 hr
+Actual time:
```
Comment (HenevianAcero) [inline prac_07/project.py:4]: Missing actual time completed

File: prac_07/project_management.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
+import datetime
+import operator
```
Comment (HenevianAcero) [inline prac_07/project_management.py:2]: Missing docstring

File: prac_07/project_management.py [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    in_file = open(file_name, 'r')
+    in_file.readline()
+    for line in in_file:
+        parts = line.strip().split("\t")
+        date = datetime.datetime.strptime(parts[1], "%d/%m/%Y").date()
+        project = Project(parts[0], date, int(parts[2]), float(parts[3]), int(parts[4]))
+        projects.append(project)
+    in_file.close()
+
+
```
Comment (HenevianAcero) [inline prac_07/project_management.py:127]: Good work

File: prac_07/myguitars.py (hunk 1 above)
Comment (alexdamrow) [reply prac_07/myguitars.py:44]: Thanks

File: prac_07/project.py (hunk 3 above)
Comment (alexdamrow) [reply prac_07/project.py:4]: Good catch, must have slipped my mind :)

File: prac_07/project_management.py (hunk 4 above)
Comment (alexdamrow) [reply prac_07/project_management.py:2]: Thanks, fixed

File: prac_07/project_management.py (hunk 5 above)
Comment (alexdamrow) [reply prac_07/project_management.py:127]: Thanks

File: prac_07/programming_language.py (hunk 2 above)
Comment (alexdamrow) [reply prac_07/programming_language.py:4]: Thanks, fixed

**Pull request merged**